data.sql

```diff
@@ -22,3 +22,33 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Ditto', '2022-05-14', 4, 'true', 22.00);
 
+
+
+-- Insert data into owners table
+INSERT INTO owners (full_name, age) VALUES
+  ('Sam Smith', 34),
+  ('Jennifer Orwell', 19),
+  ('Bob', 45),
+  ('Melody Pond', 77),
+  ('Dean Winchester', 14),
+  ('Jodie Whittaker', 38);
+
+-- Insert data into species table
+INSERT INTO species (name) VALUES
+  ('Pokemon'),
+  ('Digimon');
+
+
+BEGIN;
+UPDATE animals SET species_id=2 WHERE name LIKE '%mon';
```

Comment:
[OPTIONAL] Good job 👍🏼  , and I want just to suggest retreiving the id using select rather than using a hardcoded value this way you make your queries reusable 

```sql
  UPDATE animals
   SET species_id = (SELECT id from species WHERE name = 'Digimon')
   WHERE name like '%mon';
```